packages/core/src/external/carequality/pd/get-statistics.ts

```diff
@@ -0,0 +1,119 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { inboundPatientResourceSchema } from "@metriport/ihe-gateway-sdk";
+import { QueryTypes } from "sequelize";
+import z from "zod";
+import { MPIMetriportAPI } from "../../../mpi/patient-mpi-metriport-api";
+import { executeAsynchronously } from "../../../util/concurrency";
+import { initSequelizeForLambda } from "../../../util/sequelize";
+import { mapPatientResourceToPatientData } from "./process-inbound-pd";
+
+const MAX_NUMBER_OF_PARALLEL_XCPD_PROCESSING_REQUESTS = 20;
+
+export const rowWithDataSchema = z.object({
+  status: z.string(),
+  data: z
+    .object({
+      patientResource: inboundPatientResourceSchema.optional(),
+      timestamp: z.string(),
+    })
+    .optional(),
+});
+
+type QueryReplacements = {
+  cxId: string;
+  dateString?: string;
+  patientId?: string;
+};
+
+/**
+ * Returns statistics for XCPD, including the following:
+ * 1) # of PDs
+ * 2) # of success responses
+ * 3) # of parsed patients
+ * 4) # of mpi matches
+ *
+ * @param apiUrl        The URL of the API.
+ * @param sqlDBCreds    The SQL database credentials.
+ * @param cxId          The CX ID.
+ * @param dateString    The date string.
```

Comment:
This comment is not helpful, instead let's indicate what is this used for, please. Like, if set it will filter results only for the provided date?